manifests/04-rbac-rolebinding-cluster.yaml

```diff
@@ -39,3 +39,18 @@ subjects:
   - kind: ServiceAccount
     name: console-operator
     namespace: openshift-console-operator
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: console
```

Comment:
```suggestion
  name: console-customresourcedefinitions-reader
```

Reply:
Sam preferred to keep it console so that we could add additional permissions later. You can see his comment above.